### PR TITLE
Preload prs merged facts

### PR DESCRIPTION
**Pull request by se7entyse7en**
Closes [DEV-2312](https://athenianco.atlassian.net/browse/DEV-2312).

Depends on #1434.

### CODE REVIEW

File: server/athenian/api/experiments/preloading/entries.py [hunk 1]
```diff
@@ -112,7 +112,108 @@ class PreloadedReleaseToPullRequestMapper(ReleaseToPullRequestMapper):
...
+        assert time_to.tzinfo is not None
+        if exclude_inactive:
+            assert time_from is not None
+        # log = logging.getLogger("%s.load_merged_unreleased_pull_request_facts" %
+        #                         metadata.__package__)
+        # ghmprf = GitHubMergedPullRequestFacts
+        # postgres = pdb.url.dialect in ("postgres", "postgresql")
+        # selected = [ghmprf.pr_node_id,
+        #             ghmprf.repository_full_name,
+        #             ghmprf.data,
```
Comment (vmarkovtsev) [inline server/athenian/api/experiments/preloading/entries.py:149]: This looks suspicious :smile:

File: server/athenian/api/experiments/preloading/entries.py [hunk 2]
```diff
@@ -109,6 +109,113 @@ class PreloadedReleaseToPullRequestMapper(ReleaseToPullRequestMapper):
...
+        #             ghmprf.author,
+        #             ghmprf.merger,
```
Comment (vmarkovtsev) [inline server/athenian/api/experiments/preloading/entries.py:151]: This looks suspicious :smile:

Comment (se7entyse7en) [reply server/athenian/api/experiments/preloading/entries.py:151]: HAHAHAHAHA damn 😂
I got confused with the different branches and forgot that this one was not ready 😂😂😂

**Comment by vmarkovtsev** [comment]
:sob:

**Pull request closed without merging**